### DISCUSSION TITLE
How to change the color of the activity indicator in the Searchbar component?

### DISCUSSION THREAD

**Discussion by WillOrtiz7** [callstack/react-native-paper#3640] [Q&A]
I'm using the `Searchbar` component and I want to change the color of the activity indicator which is visible only when the `loading` prop of the `Searchbar` component evaluates to true. There is an `iconColor` prop but this only controls the color of the search icon on the left and the clear icon on the right, it does not apply to the activity indicator. Any idea what prop I can use to change the color of the activity indicator?

https://callstack.github.io/react-native-paper/searchbar.html

**Comment by csisrobi** [comment]
Did you find any solution for this? I'm stuck with this.

**Reply by ljh-wd** [reply]
Hey, having the same issue, did you find anything either?

**Comment by gedu** [comment]
Hey, at least for now there isn't a direct way to change it. PR is open to add it

As a workaround you can wrap the `Searchbar` with a new Provider, and set the color you want to color.primary
```
const themeJustForSearchbar = {
  ...DefaultTheme,
  colors: {
    ...DefaultTheme.colors,
    primary: 'red', // Customize the primary color here
  },
};

<PaperProvider theme={themeJustForSearchbar}>
       <Searchbar
              placeholder="Search"
              onChangeText={setSearchQuery}
              value={searchQuery}
              loading
            />
        </PaperProvider>
        ```

**Comment by ljh-wd** [comment]
Perfect, thank you so much!
